src/components/app/DocumentList.js

```diff
@@ -186,7 +186,9 @@ class DocumentList extends Component {
             this.setState({
                 cachedSelection: null
             }, () => {
-                dispatch(setListIncludedView());
+                if (includedView.windowType === 'pickingSlot') {
```

Comment:
for the records: this PR was closed because the fix was provided on https://github.com/metasfresh/metasfresh-webui-frontend/pull/1069